index.html

```diff
@@ -119,9 +125,9 @@ <h3 class="Section-title">My Recent Works</h3>
          </div>
       </div>
    </section>
-   <section class="about-myself">
-      <div>
-      <h3>About My Self</h3>
+   <section id="about-myself">
+      <div class="about-intro">
+      <h3>About Me</h3>
```

Comment:
Kindly implement the changes request the previous reviewer.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/173044650-a6352b2e-6b05-4c09-8874-d99dfbde1090.png) | ![image](https://user-images.githubusercontent.com/33268927/173044712-fd12123a-191c-4699-80a9-83d8d73e25a7.png) |

Reply:
@muneebulrehman this change is implemented the button is lined up already 
pls verify !